tests/js/common/shell/multi/shell-validation.js

```diff
@@ -159,23 +161,35 @@ function ValidationBasicsSuite () {
     },
 
     testAQLInsertGood: () => {
-      db._query(`INSERT { "numArray": [1, 2, 3, 4] } INTO ${testCollectionName}`);
+      db._query(`INSERT
```

Comment:
Please please please try to make no changes to unrelated code in PRs.
Reformatting unrelated code may be nice, but it makes reviewing so much harder for reviewers, especially if one-liners get broken up to multi-liners and cannot be automatically hidden by "Hide whitespace changes".